src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import CodeMirror, { Text } from '@uiw/react-codemirror'
+import { Assembly } from './assembly'
+
+interface CodeEditorState {
+  codeRunning: boolean
+}
+
+export class CodeEditor extends React.Component<{}, CodeEditorState> {
+  private editorContent: Text = Text.of([''])
+
+  state: CodeEditorState = {
+    codeRunning: false
+  }
+
+  handleCompileAndReset = (): void => {
+    // TODO: call reset() function on processor
+  }
+
+  handleRunOrHalt = (): void => {
+    if (this.state.codeRunning) {
+      // TODO: call halt() function on processor
+    } else {
+      // TODO: call execute() function on processor
+    }
+
+    // access the content of the editor
+    // see https://codemirror.net/6/docs/ref/#text
+    console.log(this.editorContent.toString())
```

Comment:
Maybe we should not commit 'console.log' calls which were mainly used for debugging and testing.

Reply:
This was more intended for other team members to show how to access the editorContent as a string. The idea was that this would be removed as soon as the content is passed over to the board for processing.

Reply:
I have remove the `console.log` statements